core/ansi_escape_parser.js

```diff
@@ -105,22 +109,25 @@ function ANSIEscapeParser(options) {
                         self.column = 1;
                     }
 
-                    self.emit('literal', text.slice(start, pos));
-                    start = pos;
+                    self.emit('literal', text.slice(start, pos + 1));
+                    start = pos + 1;
 
                     self.row += 1;
 
                     self.positionUpdated();
                     break;
 
                 default:
-                    if (self.column === self.termWidth) {
+                    if (self.column === self.breakWidth) {
                         self.emit('literal', text.slice(start, pos + 1));
                         start = pos + 1;
 
+                        // If the art is terminal, then we need to force the terminal to go to the next line.
```

Comment:
I'm having trouble understanding the comment (though I understand the code)

Reply:
I a word. Fixed :)